resources/building-mobile-extensions/building-an-extension.md

```diff
@@ -31,8 +31,12 @@ To create a simple extension, complete the following procedures in the order in
 
 ### A. **Create an extension class**
 
+
 The `ACPExtension`\(iOS\) or `Extension` \(Android\) class is the base class from which extensions must be derived. The `init` method \(iOS\) or the base `constructor` \(Android\) of your extension class is where you can extend the Adobe Experience Platform SDKs functionality by registering event listeners or by setting a default shared state that other modules can access.
 
+The Adobe Experience Cloud Platform SDK  will  lowercase the strings  like extension name, event type and event source strings, and  it is recommended  to compare the extension name, event type and event source strings using ignore case whenever compared.
```

Comment:
Please change the sentence so that it reads:

"In the Adobe Experience Cloud Platform SDK, strings like the extension name, the event type, and the event source strings are in lower case. You should compare these strings by using ignore case."

- What is "ignore case"? If this an API, please use the code syntax.

- Looks like there are gaps between words that need to be removed.